connectproto/buf.gen.yaml

```diff
@@ -0,0 +1,11 @@
+version: v1
+plugins:
+  - name: go
+    out: pb
+    opt: paths=source_relative
+  - name: go-grpc
+    out: pb
+    opt: paths=source_relative
+  - plugin: connect-go
```

Comment:
We need to compile our connectrpc stuff with this plugin - so for now I think separate buf modules are the cleanest way to do this. I think ideally it's all under `proto`, so we'd have a dir structure like `proto/connect` and `proto/everythingelse`, but if we do a move in this PR it's going to make the diff huge so I made them both top-level dirs for this PR and I think we should do the move in a follow-up.